pwiz_tools/Skyline/Properties/Resources.Designer.cs

```diff
@@ -365,7 +365,7 @@ public class Resources {
         }

```

Comment:
Unfortunate amount of churn here, hopefully a one time effect?

Reply:
Unless you are sure this will not get switched back by the next developer, please revert the lines unrelated to your changes.